frontend/language/src/nb.json

```diff
@@ -1295,6 +1295,8 @@
   "ux_editor.component_properties.subform.choose_layout_set": "Velg sidegruppe...",
   "ux_editor.component_properties.subform.choose_layout_set_label": "Velg sidegruppe å knytte til underskjema",
   "ux_editor.component_properties.subform.go_to_layout_set": "Gå til utforming av underskjemaet",
+  "ux_editor.component_properties.subform.layout_set_is_missing_content_heading": "Underskjemaet ditt mangler innhold.",
+  "ux_editor.component_properties.subform.layout_set_is_missing_content_paragraph": "Denne tabellen bruker underskemaet for å hente feltene og tekstene som skal vises i tabellen. Velg Utform underskjemaet for å legge inn innhold.",
```

Comment:
Just missing one letter in `underskemaet` => `Underskjemaet`

Reply:
Oh, good catch!